src/foam/crypto/hash/HashingJDAO.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2017,2018 The FOAM Authors. All Rights Reserved.
```

Comment:
Copyright should just be the first year of creation (according to Google lawyers) and doesn't need to ever be updated after that.

Reply:
Fixed